crm_lead_code/__manifest__.py

```diff
@@ -22,9 +21,10 @@
 
 {
     "name": "Sequential Code for Leads / Opportunities",
-    "version": "10.0.1.0.0",
+    "version": "12.0.1.0.0",
     "category": "Customer Relationship Management",
-    "author": "OdooMRP team, "
+    "author": "MuK IT GmbH, "
+              "OdooMRP team, "
               "AvanzOSC, "
               "Serv. Tecnol. Avanzados - Pedro M. Baeza, "
```

Comment:
Please switch to my new company, `Tecnativa`, instead

Reply:
https://github.com/OCA/crm/compare/3de96a73e5319c27372006aeca3c79db42804a91..95a25015a52ec101b840f55ac7e8d36dec32234c

Reply:
Thanks! You can remove as well "Serv. Tecnol...", as it's the old company.

Reply:
https://github.com/OCA/crm/commit/81893f1c15023e12d96a9d5cb18876686540743e

Reply:
https://github.com/OCA/crm/compare/95a25015a52ec101b840f55ac7e8d36dec32234c..81893f1c15023e12d96a9d5cb18876686540743e